Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "Mux_Stats_Google_IMA",
+    products: [
+        .library(name: "Mux_Stats_Google_IMA", targets: ["Mux_Stats_Google_IMA", "GoogleInteractiveMediaAds",]),
+    ],
+    dependencies: [
+        .package(
+            name: "MUXSDKStats",
+            url: "https://github.com/muxinc/mux-stats-sdk-avplayer.git",
+            from: "2.13.0"
+        ),
+    ],
+    targets: [
+        .target(
+            name: "Mux_Stats_Google_IMA",
+            dependencies: ["MUXSDKStats", "GoogleInteractiveMediaAds"],
+            path: "MUXSDKImaListener/Classes"
+        ),
+        .binaryTarget(
+            name: "GoogleInteractiveMediaAds",
```

Comment:
afaik The Google SDK doesn't support SPM so I added it as a binary target. This means anyone using out SDK will get exactly this binary which I find to be non ideal. Bc then they cannot update unless we update our package. 

But I haven't found a good way around this, it might be the best we can do if we want to support SPM 🤔